_posts/2015-03-27-ApiBlueprint-button.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Sharing is caring - socialize you're public API blueprint
+excerpt: Learn the art of API blueprint through socialization and collaboration.
+layout: post
+date: 2015-03-27 00:00:00 +0100
+author: vincenzo
+published: true
+commets: true
+---
+
+Have you ever stumbled upon an API that was so beautifully documented and wondered, how did they do that? Well, at Apiary we believe that effective learning happens through socialization and collaboration. To help support this belief and the inspiration that stems from beautifully documented APIs, we added the ability to socialize blueprints
+
+Whenever you're surfing a public blueprint, a new **API Blueprint** button will appear in header bar. Clicking this button will open up a new window with readable markdown, that can be easily grabbed, studied, modified and republished.
```

Comment:
Now, with public documentation, you have the ability to view the underlying blueprint by clicking the API blueprint button found within the header. Clicking this button will open a new tab with readable markdown that you can learn from. Sharing knowledge is amazing!